registry/handlers/app.go

```diff
@@ -514,11 +514,12 @@ func (app *App) configureRedis(configuration *configuration.Configuration) {
 				}
 			}
 
-			conn, err := redis.DialTimeout("tcp",
+			conn, err := redis.Dial("tcp",
 				configuration.Redis.Addr,
-				configuration.Redis.DialTimeout,
-				configuration.Redis.ReadTimeout,
-				configuration.Redis.WriteTimeout)
+				redis.DialConnectTimeout(configuration.Redis.DialTimeout),
+				redis.DialReadTimeout(configuration.Redis.ReadTimeout),
+				redis.DialWriteTimeout(configuration.Redis.WriteTimeout),
+				redis.DialUseTLS(configuration.Redis.TLS.Enabled))
```

Comment:
After updating to the maintained version of redigo (gomodule/redigo), `DialTimeout` was deprecated and instructed to use `Dial` instead. The only real change here is the addition of `DialUseTLS`.